docs/challenge/index.rst

```diff
@@ -0,0 +1,108 @@
+ICML 2023 Topological Deep Learning Challenge
+=====================
+Welcome to the ICML 2023 Topological Deep Learning Challenge, hosted by the second annual `Topology and Geometry (TAG) in Machine Learning Workshop <https://www.tagds.com/events/conference-workshops/tag-ml23>`_ at ICML. 
+
+Lead organizers: Mathilde Papillon, Dr. Tegan Emerson, Dr. Henry Kvinge, Dr. Tim Doster, Dr. Bastian Rieck, Dr. Sophia Sanborn and Dr. Nina Miolane.
+
+
+Description of the Challenge
+-------------
```

Comment:
Underline needs to cover the whole title.

Reply:
the compiler seems to extend the line all the way through the page, regardless of how long it is in the .rst file (see "Preview")